locales/en/landing.ftl

```diff
@@ -39,6 +39,18 @@ learn-more-mission = Learn more about our mission
 top-questions-about-monitor = Top questions about {-brand-fx-monitor}
 see-all-faq = See all FAQs
 
+# question and answer
+what-is-breach = What exactly is a data breach?
+when-info-exposed = A data breach happens when personal or private information gets exposed, stolen or copied without permission. These security incidents can be a result of cyber attacks to websites, apps or any database where people's personal information resides. A data breach can also happen by accident like if someone's login credentials accidentally get posted publicly.
```

Comment:
The linter should have run in this PR :-( I'll file a separate PR to fix it.

There are issues with apostrophes in all the strings (`'`-> `’`).

Reply:
Thank you